tasks/delete_app.yml

```diff
@@ -11,9 +11,9 @@
     # https://github.com/ansible/ansible/issues/42302
     - { kind:  "ReplicationController", api_version: "v1" }
     - { kind:  "StatefulSet", api_version: "v1" }
-    - { kind:  "Pod", api_version: "v1" }
     - { kind:  "Job", api_version: "v1" }
     - { kind:  "CronJob", api_version: "v1beta1" }
+    - { kind:  "Pod", api_version: "v1" }
```

Comment:
this PR is beautiful :star_struck:  add a comment to secure it?

Reply:
Thanks! :slightly_smiling_face: 

I'll add a comment.
If you don't mind, i'll also add a commit to bump arnold version in the same PR to avoid running the CI twice.

Reply:
@sampaccoud Done !